Mathlib/Tactic/Have.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright (c) 2022 Arthur Paulino. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino
+-/
+import Lean
+
+def Lean.Parser.Term.haveIdLhs' :=
+  optional (ident >> many (ppSpace >>
+    checkColGt "expected to be indented" >>
+    (simpleBinderWithoutType <|> bracketedBinder))) >> optType
+
+namespace Mathlib.Tactic
+
+open Lean Elab.Tactic Meta Parser.Term
+
+syntax "have " haveIdLhs' : tactic
```

Comment:
```suggestion
syntax "have" (ident)? optType : tactic
```
We don't need the binders for the port (and they are not supported in this implementation anyhow) so we could just keep this simple.

Reply:
That will require mathport changes since it uses this syntax explicitly IIRC

Reply:
Also you still need `colGt` in the optional ident because it is a trailing identifier. Otherwise
```
have
simp
```
does the wrong thing.

Reply:
(Although I would prefer that we just support binders in `have`. I have long wanted this feature.)

Reply:
Should I revert it back to the previous syntax?